core/src/main/java/com/megaease/easeagent/core/Bootstrap.java

```diff
@@ -141,8 +133,6 @@ private static void initHttpServer(Configs conf) {
         if (port == null) {
             port = DEF_AGENT_SERVER_PORT;
         }
-        String portStr = System.getProperty(AGENT_SERVER_PORT_KEY, String.valueOf(port));
-        port = Integer.parseInt(portStr);
```

Comment:
The reason for reading from environment variables in Bootstrap.java is to enhance code readability because this variable override is very explicit. The original code can be clearly identified at this point, so please do not make changes here.